### PR TITLE
Fix CouchDB search duplicates - Mct4667

### PR DESCRIPTION
**Pull request by scottbell**
### All Submissions:

* [X] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [X] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [X] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [X] Changes address original issue?
* [X] Unit tests included and/or updated with changes?
* [X] Command line build passes?
* [X] Has this been smoke tested?
* [X] Testing instructions included in associated issue?

### CODE REVIEW

**Comment by scottbell** [comment]
To test:
1. Configure OpenMCT with CouchDB for persistence
2. Create a Clock called "Blue Clock"
3. Search for "blue"
4. Ensure you only have one result!

**Comment by scottbell** [comment]
This PR resolves:
https://github.com/nasa/openmct/issues/4667

File: src/plugins/persistence/couch/pluginSpec.js [hunk 1]
```diff
@@ -262,7 +262,9 @@ describe('the plugin', () => {
             await Promise.all(openmct.objects.search('test'));
             const requestUrl = fetch.calls.mostRecent().args[0];
 
-            expect(fetch).toHaveBeenCalled();
+            // we only want one call to fetch, not 2!
+            // see https://github.com/nasa/openmct/issues/4667
+            expect(fetch).toHaveBeenCalledTimes(1);
```
Comment (unlikelyzero) [inline src/plugins/persistence/couch/pluginSpec.js:267]: nice

**Review by unlikelyzero — APPROVED** [review]
Will need a developer more familiar with couchdb to review. @shefalijoshi or @jvigliotta ?

**Review by jvigliotta — APPROVED** [review]
Nice work! Will be glad when we fix the underlying issue so we can mash these together!

### Reviewer Checklist
Check | Passed
-- | --
Changes appear to address issue?? | Y
Appropriate unit tests included? | Y
Code style and in-line documentation are appropriate? | Y
Commit messages meet standards? | Y
Has associated issue been labelled unverified? (if PR closes issue) | Y

**Pull request merged**